quirks/password-rules.json

```diff
@@ -392,6 +392,9 @@
     "live.com": {
         "password-rules": "minlength: 8; required: lower; required: upper; required: digit; allowed: [-@_#!&$`%*+()./,;~:{}|?>=<^'[]];"
     },
+    "lloydsbank.co.uk" : {
+        "passwordrules" : "minlength: 8; maxlength: 15; required: lower; required: digit; allowed: upper"
```

Comment:
Can you toss a trailing ; here?

Reply:
Thanks! Oops, sorry I missed that, i have added it.